src/lib.rs

```diff
@@ -1357,11 +1357,15 @@ impl Build {
 
         cmd.arg(&src);
 
-        // On MSVC skip the CRT by setting the entry point to `main`.
-        // This way we don't need to add the default library paths.
         if compiler.is_like_msvc() {
-            // Flags from _LINK_ are appended to the linker arguments.
-            cmd.env("_LINK_", "-entry:main");
```

Comment:
Do we no longer need this?

Reply:
Yeah, we're including the CRT so the normal entry point will be there now.